packages/scoutgame/src/builderNfts/handleEmailFailedTransaction.ts

```diff
@@ -0,0 +1,49 @@
+import { log } from '@charmverse/core/log';
+import { prisma } from '@charmverse/core/prisma-client';
+import { sendEmailTemplate } from '@packages/mailer/sendEmailTemplate';
+import { getUser } from '@packages/nextjs/session/getUser';
+
+export async function handleEmailFailedTransaction({
+  userId,
+  pendingTransactionId,
+  errorMessage = 'Decent transaction failed'
+}: {
+  userId: string;
+  pendingTransactionId: string;
+  errorMessage?: string;
+}): Promise<void> {
+  try {
+    const pendingTx = await prisma.pendingNftTransaction.findUnique({
+      where: {
+        id: pendingTransactionId
+      }
+    });
+
+    if (!pendingTx) {
+      return;
+    }
+
+    const user = await getUser(userId);
```

Comment:
its not necessary as we already fetch the user inside `sendEmailTemplate`

Reply:
I need it for the `name` variable